egg-herbie/main.rkt

```diff
@@ -22,7 +22,40 @@
                  [else "libegg_math"])
                 (bytes->string/utf-8 (system-type 'so-suffix)))))
 
-(define-ffi-definer define-eggmath (ffi-lib libeggmath-path))
+; Checks if Racket is being run in emulation via rosetta
+(define (running-on-rosetta?)
+  (if (and (equal? (system-type 'os) 'macosx)

```

Comment:
Why do an `if` where the else branch is `false`? It should just be part of the `and`